### PR TITLE
Update 009-git-workflow.md

### PR DESCRIPTION
**Pull request by lfrohlich**
### Summary of Changes
Added 'ready-to-merge' label and GitNotify channel in Teams to relevant steps. Updated reviewers. Removed security controls.

### CODE REVIEW

File: docs/Architecture Decision Record/009-git-workflow.md [hunk 1]
```diff
@@ -50,21 +49,21 @@ A contributor to the TDP project would always use the following steps to propose
...
+     * posts that PR is ready for QASP review in GitNotify channel in Teams and tags appropriate government reviewer.
+18. For PRs with `qasp-review` label, the appropriate government reviewer performs the review and/or requests changes. **GOAL** 5 days: 3 days to perform the review and 2 days to implement the requested changes.
     * When changes are asked for, the changes are made by the contributor. Raft reviewers should internalize the changes asked by the Government such that the same feedback is already incorporated and/or caught in future (continuous improvement)
     * When security controls are approved by the Government, the Government reviewer will ask for additional reviews to be performed by OCIO
-19. When satisfied, the Government reviewer `approves` the PR
+19. When satisfied, the Government reviewer `approves` the PR and tags with the  `ready-to-merge` label. Government reviewer posts that PR is ready to merge in GitNotify channel on Teams.   
 20. The contributor clicks Merge into `raft-tdp-main`
```
Comment (lfrohlich) [inline docs/Architecture Decision Record/009-git-workflow.md:61]: should this be tech lead?

File: docs/Architecture Decision Record/009-git-workflow.md [hunk 2]
```diff
@@ -50,21 +49,21 @@ A contributor to the TDP project would always use the following steps to propose
...
         * For research and design work assign `lfrohlich`
-        * For security controls assign `lfrohlich`
-18. For PRs with `qasp-review` label, the appropriate reviewer performs the review and/or requests changes. **GOAL** 5 days: 3 days to perform the review and 2 days to implement the requested changes.
+     * posts that PR is ready for QASP review in GitNotify channel in Teams and tags appropriate government reviewer.
+18. For PRs with `qasp-review` label, the appropriate government reviewer performs the review and/or requests changes. **GOAL** 5 days: 3 days to perform the review and 2 days to implement the requested changes.
     * When changes are asked for, the changes are made by the contributor. Raft reviewers should internalize the changes asked by the Government such that the same feedback is already incorporated and/or caught in future (continuous improvement)
     * When security controls are approved by the Government, the Government reviewer will ask for additional reviews to be performed by OCIO
```
Comment (lfrohlich) [inline docs/Architecture Decision Record/009-git-workflow.md:59]:
```suggestion

```

File: docs/Architecture Decision Record/009-git-workflow.md [hunk 3]
```diff
@@ -50,21 +49,20 @@ A contributor to the TDP project would always use the following steps to propose
...
+     * posts that PR is ready for QASP review in GitNotify channel in Teams and tags appropriate government reviewer.
+18. For PRs with `qasp-review` label, the appropriate government reviewer performs the review and/or requests changes. **GOAL** 5 days: 3 days to perform the review and 2 days to implement the requested changes.
     * When changes are asked for, the changes are made by the contributor. Raft reviewers should internalize the changes asked by the Government such that the same feedback is already incorporated and/or caught in future (continuous improvement)
-    * When security controls are approved by the Government, the Government reviewer will ask for additional reviews to be performed by OCIO
-19. When satisfied, the Government reviewer `approves` the PR
+19. When satisfied, the Government reviewer `approves` the PR and tags with the  `ready-to-merge` label. Government reviewer posts that PR is ready to merge in GitNotify channel on Teams.   
 20. The contributor clicks Merge into `raft-tdp-main`
```
Comment (ADPennington) [inline docs/Architecture Decision Record/009-git-workflow.md:60]:
```suggestion
20. `abottoms-coder` clicks Merge into `raft-tdp-main`
```

File: docs/Architecture Decision Record/009-git-workflow.md [hunk 4]
```diff
@@ -50,21 +49,20 @@ A contributor to the TDP project would always use the following steps to propose
...
 21. The contributor:
```
Comment (ADPennington) [inline docs/Architecture Decision Record/009-git-workflow.md:61]:
```suggestion
21. `abottoms-coder` (or his back-up):
```

**Review by ADPennington — COMMENTED** [review]
@abottoms-coder @jtwillis92, this is ready for your review. Feel free to at boilerplate language re:  dependency review on the raft side + gov side (we can also wait for this until the dependabot ADR update has been reviewed and approved).

**Review by abottoms-coder — APPROVED** [review]

**Pull request merged**